d2bs/kolbot/libs/bots/BattleOrders.js

```diff
@@ -5,6 +5,43 @@
 */
 
 function BattleOrders() {
+	this.AmTardy = function() {
+		let party;
+
+		if (Config.BattleOrders.SkipIfTardy) {
```

Comment:
This is fine of course, but as there is an entire block of indentation, it might be more readable if you write, where you call this function, it like this.

```if (Config.BattleOrders.SkipIfTardy && this.AmTardy()) { 
    break MainLoop;
}
```

Or, here write it like

```
if (!Config.BattleOrders.SkipIfTardy) {
   return false; // Never late if we dont care
}
// The original code

return false; // Not late
```